docs/v4/mini-faq.md

```diff
@@ -0,0 +1,198 @@
+# Mini-FAQ
+
+@jesus-seijas-sp collected a mini FAQ for some v4 topics:
+
+Hi,
+
+I will put here a FAQ with the links to different interesting parts to documentation or examples. If there is something that you want to cover that is not here, just ask, and I will update the documentation with it. Also, there are 1800 unit tests that can help to understand classes and functions that are not intended to be the API for the developer (from node-nlp the API for the developer is intented to be the class NlpManager).
+
+**- Where do I find an example of use of v4?** https://github.com/axa-group/nlp.js#example-of-use
```

Comment:
Good but we can also add a link here to [examples folder](https://github.com/axa-group/nlp.js/tree/master/examples) to help user understand what it covers.